### DISCUSSION TITLE
language statistics

### DISCUSSION THREAD

**Discussion by iibrahemali** [anuraghazra/github-readme-stats#3929] [Q&A] [answered]
**does the language usage in a private repo contribute to the language stats on the language card?**

**Comment by lulunac27a** [comment, marked as answer]
According to [Top Languages Card](https://github.com/anuraghazra/github-readme-stats#top-languages-card), you need to deploy your own instance using your GitHub API token.

**Reply by qwerty541** [reply]
You're right, displaying private contributions requires own instance deployment using own API token. I will just leave there a link to the documentation with step by step guide: https://github.com/anuraghazra/github-readme-stats?tab=readme-ov-file#deploy-on-your-own
